### ISSUE TITLE
Request: Add custom set prices for individual items

### ISSUE THREAD

**Issue by elrufa** [Roeshambo/MonolithDKP#139]
My guild uses custom prices for each item. We have assigned certain values per slot in the addon optinos, but we have several pieces that we would like to change the values up or down from their "slot-value". Is it possible to add options to basically price each item drop individually. 

I don't mind if this is just inside a settings lua file or something if making a GUI for this is cumbersome, but it would be great to have the option instead of having to do it manually on the bid frame when an item drops. Even if I have a sheet with all the prices on the side it's easy to forget sometimes and having to adjust afterwards can be kinda awkward.

Thanks!

**Comment by Roeshambo** [comment]
You can actually already do that. In the bid frame when you're auctioning an item off, set the desired price. It'll save that price and use that number the next time that specific item is loaded into the window

**Comment by elrufa** [comment]
> You can actually already do that. In the bid frame when you're auctioning an item off, set the desired price. It'll save that price and use that number the next time that specific item is loaded into the window

Yeah I've seen that feature. I was just just wondering if it would be possible to set the values in advance for each items so I don't have to reference my sheet whenever an item drops. Or if you can point me towards where these values are saved I could maybe add them myself in the files. Thanks!

**Comment by Roeshambo** [comment]
There's no way to do it within the addon itself. But you can add them to your saved variables file. Do a search for MonDKP_MinBids. the format for it is:

```
{
        ["item"] = "Item Name",
        ["minbid"] = amount,
},
{
        ["item"] = "Striker's Mark",
        ["minbid"] = 75,
},
```
